pkg/util/intstr/intstr.go

```diff
@@ -44,7 +44,7 @@ const (
 
 // FromInt creates an IntOrString object with an int value.
```

Comment:
Add: it's your responsibility to not call this with an int that won't fit in 32 bits.